stdlib/test/builtin/test_simd.mojo

```diff
@@ -1546,6 +1546,41 @@ def test_contains():
     assert_false(0 in y or 5 in y)
 
 
+def test_comparison():
+    alias I = SIMD[DType.int32, 4]
+    var i = I(-2, 0, 2, 10)
```

Comment:
Could you please also add some tests for bool and a floating point type? I think `test_truthy()` might give you some inspiration on how to do expand coverage for data types.

Reply:
thanks for the pointer, will check that test.

Reply:
addressed this in [60afbc7](https://github.com/modularml/mojo/pull/3221/commits/60afbc7ac33a406de178002660a561dcad6f462a)